<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Selection</class>
 <widget class="QDockWidget" name="Selection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Selection</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="selection_mode_horizontalGroupBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Selection mode</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <layout class="QGridLayout" name="gridLayout_6">
               <item row="1" column="0">
                <widget class="QLabel" name="Smart_label">
                 <property name="text">
                  <string>Mode: </string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="selection_mode_label">
                 <property name="text">
                  <string>Selection type:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QRadioButton" name="select_facet_radioButton">
                 <property name="text">
                  <string>Triangles</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">buttonGroup</string>
                 </attribute>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="Smart_type_combo_box">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                   <underline>false</underline>
                   <strikeout>false</strikeout>
                   <stylestrategy>PreferDefault</stylestrategy>
                   <kerning>false</kerning>
                  </font>
                 </property>
                 <item>
                  <property name="text">
                   <string>none</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Extract small parts with a lasso</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Split non-planar segment with a stroke</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Split planar segment with region growing</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QRadioButton" name="select_segment_radioButton">
                 <property name="text">
                  <string>Segments</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">buttonGroup</string>
                 </attribute>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="basic_selection_groupbox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Basic selection</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="3">
                <widget class="QPushButton" name="Expand_reduce_button">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Expand/Reduce</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="4">
                <widget class="QSpinBox" name="Expand_reduce_spin_box">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimum">
                  <number>-50</number>
                 </property>
                 <property name="maximum">
                  <number>50</number>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSpinBox" name="Brush_size_spin_box">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
               <item row="0" column="5">
                <widget class="QCheckBox" name="lassoCheckBox">
                 <property name="text">
                  <string>Lasso</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="selection_rings_label">
                 <property name="text">
                  <string>Multi-selection rings:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QPushButton" name="SelectClasses">
                 <property name="text">
                  <string>Select Semantic Class</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="select_classes_comboBox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <item>
                  <property name="text">
                   <string>None</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QPushButton" name="Select_all_button">
              <property name="toolTip">
               <string extracomment="Select all simplices of Selection Type"/>
              </property>
              <property name="text">
               <string>Select &amp;All</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Inverse_selection_button">
              <property name="toolTip">
               <string extracomment="Invert selection for Selection Type"/>
              </property>
              <property name="text">
               <string>Invert Selection</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Clear_all_button">
              <property name="toolTip">
               <string extracomment="Clear selection for All Types"/>
              </property>
              <property name="text">
               <string>Clear Selection</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
